src/app/shared/search/advanced-search/advanced-search.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="facet-filter d-block mb-3 p-3" [ngClass]="{ 'focus': focusBox }" role="region">
+    <button (click)="toggle()" (focusin)="focusBox = true" (focusout)="focusBox = false" class="filter-name d-flex"
+        [attr.aria-expanded]="false" [attr.data-test]="'filter-toggle' | dsBrowserOnly">
+        <h5 class="d-inline-block mb-0">
+            {{'search.advanced.filters.head' | translate}}
+        </h5>
+        <span class="filter-toggle flex-grow-1 fas p-auto" [ngClass]="collapsedSearch ? 'fa-plus' : 'fa-minus'">
+        </span>
+    </button>
+    <div [@slide]="collapsedSearch ? 'collapsed' : 'expanded'" (@slide.start)="startSlide($event)"
+        (@slide.done)="finishSlide($event)" class="search-filter-wrapper"
+        [ngClass]="{ 'closed' : closed, 'notab': notab }">
+        <form [class]="'ng-invalid'" [formGroup]="advSearchForm" (ngSubmit)="onSubmit(advSearchForm.value)"
+            >
+            <div class="row">
+                <div class="col-lg-12">
+                    <select
+                        [className]="(filter.invalid) && (filter.dirty || filter.touched) ? 'form-control is-invalid' :'form-control'"
+                        aria-label="filter" name="filter" id="filter" placeholder="select operator" formControlName="filter" required>
+                        <ng-container *ngFor="let filter of appConfig.search.advancedFilters.filter;">
+                            <option [value]="filter">
+                                {{'search.filters.filter.' + filter + '.text'| translate}}
+                            </option>
+                        </ng-container>
+                    </select>
+                </div>
+                <div class="col-lg-12 mt-1">
+                    <input type="text" aria-label="textsearch"
+                        [className]="(textsearch.invalid) && (textsearch.dirty || textsearch.touched) ? 'form-control is-invalid' :'form-control'"
+                        id="textsearch" name="textsearch" formControlName="textsearch" #text
+                        [placeholder]="('filter.search.text.placeholder' | translate)" required>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-lg-8 mt-1">
+                    <select
+                        [className]="(operator.invalid) && (operator.dirty || operator.touched) ? 'form-control is-invalid' :'form-control'"
+                        aria-label="operator" name="operator" id="operator" formControlName="operator" required>
+                        <option value="equals">{{'search.filters.operator.equals.text'| translate}}</option>
```

Comment:
This key `search.filters.operator.equals.text` is missing from our `en.json5` file. It needs to be added.